packages/reporters/dev-server/src/Server.js

```diff
@@ -207,6 +207,7 @@ export default class Server {
       let requestedPath = path.normalize(pathname.slice(1));
       let bundle = bundleGraph
         .getBundles()
```

Comment:
cc @devongovett and @mischnic we recently discussed changing this method to never return inline bundles

Reply:
I tried to filter inline bundles in `getBundles` method, but many tests failed and I didn't figure out how to fix them. This change is definitely out of my knowledge of Parcel. However this bug is blocking my development, so should I finish this pull request as a quick fix, or you change `getBundles` method soon?

Reply:
I think we should merge this for now (as there are also various other places where we already do `getBundles().filter(b => !b.isInline)` and change `getBundles` later.